src/components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from "react";
+import styles from './TextInput.module.css';
+
+type TextProps = {
+    top?: number;
+    bottom?: number;
+}
+
+export default function({ top, bottom }: TextProps):ReactElement {
+    const margins = {
+        marginTop: 0,
+        marginBottom: 0
+    }
+    if(top) { margins.marginTop = top};
+    if(bottom) { margins.marginBottom = bottom};
+
+    return (
+        <div className={styles.container} style={margins}>
+            <span className={styles.textarea} contentEditable></span>
```

Comment:
**nitpick:** This is very smart move, however it's less accessible solution than `textarea`. Screen readers and other assistive technologies might not handle the dynamic content and formatting changes well. Another thing is that `contentEditable` is less secure as it allows you to paste rich formatted text, and `textarea` accepts string only. You will have to sanitize your pasted / inserted content to make sure you don't do eg. XSS